packages/gitlab-backend/src/plugin.ts

```diff
@@ -0,0 +1,47 @@
+import {
+    coreServices,
+    createBackendModule,
+    createBackendPlugin,
+} from '@backstage/backend-plugin-api';
+import { catalogProcessingExtensionPoint } from '@backstage/plugin-catalog-node/alpha';
+import { loggerToWinstonLogger } from '@backstage/backend-common';
+import { GitlabFillerProcessor } from './processor';
+import { createRouter } from './service/router';
+
+export const catalogPluginGitlabFilterProcessorModule = createBackendModule({
+    pluginId: 'catalog',
+    moduleId: 'gitlabFilterProcessor',
+    register(env) {
+        env.registerInit({
+            deps: {
+                config: coreServices.rootConfig,
+                extensionPoint: catalogProcessingExtensionPoint,
+            },
+            async init({ config, extensionPoint }) {
+                extensionPoint.addProcessor(new GitlabFillerProcessor(config));
+            },
+        });
+    },
+});
+
+export const gitlabPlugin = createBackendPlugin({
+    pluginId: 'gitlab',
```

Comment:
The id will used to register the router, so  it should be gitlab instead of gitlab-backend.